src/modules/apps/utils.ts

```diff
@@ -144,3 +145,39 @@ export function optionsFormatter(billingOptions: BillingOptions) {
   table.push([{ content: chalk.bold('Terms of use:'), hAlign: 'center' }, { content: billingOptions.termsURL, hAlign: 'center' }])
   return table.toString()
 }
+
+export async function checkBuilderHubMessage(cliRoute: string): Promise<any> {
+  const http = axios.create({
+    baseURL: `https://vtex.myvtex.com`,
+    timeout: 10000,
```

Comment:
How about changing this timeout depending on the route (`link` versus `publish`)?